_pages/dev/tms.md

```diff
@@ -0,0 +1,205 @@
+# Tag Management System
+
+Spartacus introduced the Tag Management System (TMS) integration in version 3.2.
+Before you continue reading this page, it's highly recommended to first get yourself familiar with the [Events Service](./event-service), as TMS relies on it.
+
+The TMS integration allows you to specify which Spartacus' events should be passed to the configured TMS. Google Tag Manager (GTM) and Adobe Experience Platform (AEP) systems are supported out of the box, while the other tag managers can easily be plugged in.
+
+Spartacus supports running multiple TMS in parallel, and you can decide which events should be collected by each of the supported TMS systems.
+
+## Setup
+
+Each TMS system might require a different setup instruction, and it's not possible to cover every system here.
+However, the process _usually_ requires you to specify a certain `<script>` tag in your `index.html` which will load and bootstrap the TMS.
+
+After this is done, we can move on to the [configuration](#configuration) section.
+
+## Configuration
+
+You can import the `TmsModule` and pass the configuration to it like this:
```

Comment:
Perhaps we start explaining why/what we would configure in the first place?

Reply:
Done